Vic2ToHoI4/Source/HOI4World/HoI4Country.h

```diff
@@ -245,6 +247,9 @@ class Country
 
 	[[nodiscard]] const bool isEligibleEnemy(std::string target);
 
+	void addGeneratedDominion(const std::string& region, const std::string& tag) { generatedDominions[region] = tag; }
+	const std::string& getDominionTag(const std::string& region);
```

Comment:
Since there may not be a dominion tag in a region, this should return std::optional<std::string> instead.